<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2024</width>
    <height>1090</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="main_camera">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>109</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera_1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>120</y>
      <width>210</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>120</y>
      <width>210</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>240</y>
      <width>210</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera_4">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>240</y>
      <width>210</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera_5">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>360</y>
      <width>210</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera_6">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>360</y>
      <width>210</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_box">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_axis">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_box">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>50</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 85, 255);</string>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_axis">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>690</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>Camera 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Camera 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Camera 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Camera 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Camera 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Camera 6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
